### PR TITLE
Adds bluespace emag as a badmin item

### PR DESCRIPTION
**Pull request by Shadowlight213**
Also Removes depreciated clown data card.
Bluespace emag ported from #16322

### CODE REVIEW

File: code/game/objects/items/weapons/cards_ids.dm [hunk 1]
```diff
@@ -63,13 +53,21 @@
 	item_state = "card-id"
 	origin_tech = "magnets=2;syndicate=2"
 	flags = NOBLUDGEON
+	var/prox_check = TRUE //If the emag requires you to be in range
+
+/obj/item/weapon/card/emag/bluespace
+	name = "bluespace cryptographic sequencer"
+	desc = "It's a blue card with a magnetic strip attached to some circuitry."
```
Comment (GunHog) [inline code/game/objects/items/weapons/cards_ids.dm]: Add "It seems to have some sort of transmitter attached" or something.

File: code/game/objects/items/weapons/cards_ids.dm [hunk 2]
```diff
@@ -63,13 +53,21 @@
 	item_state = "card-id"
 	origin_tech = "magnets=2;syndicate=2"
 	flags = NOBLUDGEON
+	var/prox_check = TRUE //If the emag requires you to be in range
+
+/obj/item/weapon/card/emag/bluespace
+	name = "bluespace cryptographic sequencer"
+	desc = "It's a blue card with a magnetic strip attached to some circuitry.It appears to have some sort of transmitter attached to it."
```
Comment (GunHog) [inline code/game/objects/items/weapons/cards_ids.dm]: Space needed there

File: code/game/objects/items/weapons/cards_ids.dm [hunk 3]
```diff
@@ -43,16 +43,6 @@
 	src.add_fingerprint(usr)
 	return
 
-/obj/item/weapon/card/data/clown
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/cards_ids.dm]: BRING BACK CLOWN PLANET

Comment (Shadowlight213) [inline code/game/objects/items/weapons/cards_ids.dm]: I asked in coderbus about removing it, and was told that it wasn't used anywhere and should be removed.

File: code/game/objects/items/weapons/cards_ids.dm (hunk 2 above)
Comment (Shadowlight213) [inline code/game/objects/items/weapons/cards_ids.dm]: REEEEE GRAMMER

File: code/game/objects/items/weapons/cards_ids.dm (hunk 3 above)
Comment (Bawhoppen) [inline code/game/objects/items/weapons/cards_ids.dm]: DON'T REMOVE I HAVE PLANS FOR IT

**Comment by Bawhoppen** [comment]
also
>not sneaking a bluespace (person) reference into it

**Comment by Shadowlight213** [comment]
In sorry, i can not do that. That would violate our reference policy.

**Comment by Niknakflak** [comment]
who wants to reference that scummy meme anyway

File: code/game/objects/items/weapons/cards_ids.dm (hunk 3 above)
Comment (KazeEspada) [inline code/game/objects/items/weapons/cards_ids.dm]: Except we have away missions and lavaland templates, if you are going to use it, just readd it. Besides, its shouldn't be a child of ID cards.

**Comment by KorPhaeron** [comment]
I mean it's admin only so whatever but it seems like "guy sits at a camera console and now the whole station is emagged" will only be funny maybe once.

**Comment by KazeEspada** [comment]
Isn't this the same as having an a rogue AI anyways?

**Pull request merged**